src/Raven.Server/RavenServer.cs

```diff
@@ -423,7 +423,7 @@ private void UpdateCertificateExpirationAlert()
                 if (Logger.IsOperationsEnabled)
                     Logger.Operations(msg);
             }
-            else if (remainingDays <= 20)
+            else if (remainingDays <= Configuration.Core.AcmeDaysToRenewBeforeExpiration)
```

Comment:
I would rather have this computed as percentage.
So use 30% (so 27 days for 90 days cert, and ~50 hours or so for 7 days cert).

That way, you don't need any special configuration to think about.